src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Don't use tag names for styling (except `html` and `body`). While it's generally acceptable to style the `body` element, the `box-sizing: border-box;` declaration is better set globally using the `*` selector. Even though the checklist advises against using the `*` selector, setting `box-sizing` is a common exception to this rule.